azure/Guidelines.md

```diff
@@ -698,7 +698,11 @@ Azure services need to change over time. However, when changing a service, there
 
 :white_check_mark: **DO** review any API changes with the Azure API Stewardship Board
 
-:white_check_mark: **DO** use an `api-version` query parameter with a `YYYY-MM-DD` date value, with a `-preview` suffix for a preview service.
+Clients specify the version of the API to be used in every request to the service, even requests to an `Operation-Location` or `nextLink` URL returned by the service.
```

Comment:
Should this explanation go below the **DO** directive right below it? IIRC, I thought we put explanations after the requirement.

Reply:
I would be okay to move it, but I think this sentence "provides context" for the guidelines below rather than "explaining" them. That's why I put it above.

Reply:
Okay. Just a nit. Works for me either way. :)